os-autoinst-openvswitch

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/perl
+package OVS;
+use strict;
+
+use base qw(Net::DBus::Object);
+use Net::DBus::Exporter qw(org.opensuse.os_autoinst.switch);
```

Comment:
I would prefer `org.opensuse.os_autoinst.openvswitch` service instead of plain switch.